src/readbin/func_start.ml

```diff
@@ -0,0 +1,36 @@
+open Bap.Std
+open Core_kernel.Std
+open Or_error
+
+exception Bad_user_input of string
+
+let bap_byteweight bin subcmd =
+  let tmp = Filename.temp_file "bw_" ".output" in
+  let cmd = Printf.sprintf "bap-byteweight %s -x %S > %S" subcmd bin tmp in
+  try
+    let _ = Unix.system cmd in
+    Symbols.read_addrset tmp
+  with _ -> raise (Bad_user_input subcmd)
+
+let byteweight bin = bap_byteweight bin "find"
+
+let symbols bin = bap_byteweight bin "symbols"
+
+let user = Symbols.read_addrset
+
+let ida bin : Addr.Set.t =
+  let sets_of_table t : Addr.Set.t =
```

Comment:
This is the same as it was previously, just under different dressing. 

This is bad:

Table => Sequence => List => Set 

This is good:

Table => Set.

No `to_list` or `of_list`. Use `Table.fold` and `Addr.Set.add`